.github/workflows/code-quality.yml

```diff
@@ -27,12 +29,16 @@ jobs:
     needs: [coding-standards, linting]
     uses: ./.github/workflows/static-analysis.yml
     secrets:
-      COMPOSER_AUTH_JSON: ${{ secrets.COMPOSER_AUTH_JSON }}
+      OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+      OP_VAULT: ${{ secrets.OP_VAULT }}
+      OP_ITEM: ${{ secrets.OP_ITEM }}
 
 # Enable this job if you have PHPUnit tests
```

Comment:
Out of curiosity, is it common to have PHP unit tests in Moose projects? Just wondering if this is something we maybe just want to remove (or move to Moose Resources). Not necessary in this PR, but wanted to note it.

Reply:
Yeah, it's definitely worth a conversation about removing for now.